Server/controllers/authController.js

```diff
@@ -237,6 +238,21 @@ const inviteController = async (req, res, next) => {
   }
 };
 
+const inviteVerifyController = async (req, res, next) => {
+  try {
+    await inviteVerifciationBodyValidation.validateAsync(req.body);
+    const invite = await req.db.getInviteToken(req, res);
+
+    res
+      .status(200)
+      .json({ status: "success", msg: "Invite verified", data: invite });
+  } catch (error) {
+    error.service = SERVICE_NAME;
+    next(error);
```

Comment:
Function will end at ```next(error);```
No need for a return here I don't think.

Reply:
I belive you are correct here, the return is left over from an earlier function I wrote that had a conditional `next()` call and I wanted to make sure no further code was executed after that so I added the `return` and now I've just gotten in the habit of doing it to be safe.